CHANGELOG.md

```diff
@@ -34,6 +34,36 @@ New features:
   Now, these types' kind signatures are displayed above their declarations
   in their docs, similar to what one would see in the source code.
 
+  But that's not all!
+
+  The two below types do not have explicit kind signatures, but the compiler
+  infers what their kind signature is anyway. "Interesting" kind signatures
+  will be displayed but "uninteresting" kind signatures (even if they are
+  explicit) will be removed.
+
+  ```purescript
+                                                  {-
+  data Foo :: (Type -> Type) -> Type -> Type      -}
```

Comment:
The compiler will infer `forall k. (k -> Type) -> k -> Type` for this data type, not `(Type -> Type) -> Type -> Type`.

Reply:
Thanks. I decided to adjust this example to use `(f Int)` rather than `(f a)`